lib/campaign/CampaignState.h

```diff
@@ -33,6 +33,8 @@ class IGameCallback;
 class DLL_LINKAGE CampaignRegions
 {
 	std::string campPrefix;
+	std::vector<std::string> campSuffix = {};
+	std::string campBackground = "";
```

Comment:
Only primitive types (bool, int's, raw pointers, enums) need default-init
```suggestion
	std::vector<std::string> campSuffix;
	std::string campBackground;
```